modules/page-interactions.js

```diff
@@ -0,0 +1,50 @@
+const dateContainer = document.querySelector('.date-container');
+const booksList = document.querySelector('#book-list');
+const addBook = document.querySelector('#addBook');
+const contact = document.querySelector('#contact');
+const navElements = Array.from(document.getElementsByClassName('nav-element'));
+
+const GeneralFunctions = {
+  init() {
+    dateContainer.innerHTML = this.date();
+    this.changeDisplay(0);
+    this.listenNavbarItems();
+  },
+  date: () => {
+    /* eslint-disable */
+    const { DateTime } = luxon;
+    /* eslint-disable */
+    const now = DateTime.now();
+    const timeSimbol = now.hour > 12 ? 'pm' : 'am';
+    const fixedDate = `${now.monthLong} ${now.day} ${now.year},${now.hour}:${now.minute}:${now.second} ${timeSimbol}`;
+    return fixedDate;
+  },
+  changeDisplay: (index) => {
+    switch (index) {
+      case 0:
+        booksList.classList.toggle('hide');
+        addBook.classList.add('hide');
+        contact.classList.add('hide');
+        break;
+      case 1:
+        addBook.classList.toggle('hide');
+        booksList.classList.add('hide');
+        contact.classList.add('hide');
+        break;
+      case 2:
+        contact.classList.toggle('hide');
+        booksList.classList.add('hide');
+        addBook.classList.add('hide');
+        break;
+      default:
+        break;
+    }
+  },
```

Comment:
- [ ]  While displaying each page you are supposed to add and remove CSS classes to display or hide the content by using the `Element.classList add ` and `remove `methods. This will make your code much cleaner and reduce repetition.
You can implement this by looping through the links and rendering the content dynamically when each link is clicked on. Checkout this [article](https://dev.to/mainakibe/creating-a-simple-horizontal-switchable-tabs-panel-using-html-css-javascript-e0l) for more information.

Reply:
I understand your suggestion and we read the article but we decided to do it in an easy way to understand and shorter than that, but thanks you so much for your advice.